external/netstandard/netstandard.depproj

```diff
@@ -6,14 +6,21 @@
     <NugetRuntimeIdentifier>None</NugetRuntimeIdentifier>
     <OutputPath>$(RefRootPath)/$(TargetGroup)</OutputPath>
     <ProjectJsonTemplate Condition="$(NuGetTargetMoniker.Contains('Version=v1.'))">$(MSBuildProjectDirectory)\netstandard1.x\project.json.template</ProjectJsonTemplate>
+    <_NETStandardTFMFolder Condition="'$(NuGetTargetMoniker)' == '.NETStandard,Version=v2.0'">netstandard2.0</_NETStandardTFMFolder>
+    <_NETStandardTFMFolder Condition="$(NuGetTargetMoniker.StartsWith('.NETFramework'))">net461</_NETStandardTFMFolder>
```

Comment:
Assuming things are correctly staged, which I believe they are I think I would prefer we overwrite instead of maintaining a filtering list.